doc/CB-to-STH-documentation/TourGuide-STH-Documentation.md

```diff
@@ -0,0 +1,227 @@
+# How to connect STH-Comet to Orion Context Broker
```

Comment:
Better title

# How to generate the history of Context Information using STH-Comet